bucket/tidal.json

```
@@ -0,0 +1,28 @@
+{
+    "version": "2.36.2",
+    "description": "A music streaming service that offers music with high fidelity sound quality",
+    "homepage": "https://tidal.com/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://tidal.com/terms"
+    },
+    "url": "https://download.tidal.com/desktop/TIDALSetup.exe#/dl.7z",
+    "hash": "d4f2f3bfb35c71e75ec4cae40bcc729a053b7913c74cc116ed9583b15c909fac",
+    "pre_install": [
+        "Expand-7zipArchive \"$dir\\TIDAL-$version-full.nupkg\" -ExtractDir 'lib\\net45' -Removal",
+        "Remove-Item \"$dir\\lib\", \"$dir\\Update.exe\""
```

```suggestion
        "Remove-Item \"$dir\\lib\"
```
This is an electron program that can get patches without version number changes (invisible minor versions).